server/core/controller.js

```diff
@@ -7,6 +7,32 @@ class Controller {
         var Model = this.model;
         var perPage = config.get('pagination.itemsPerPage');
         var page = (req.query.page) ? req.query.page : 1;
+
+        var component = this.findAllComponent(req);
```

Comment:
`findAllComponent` is confusing as a function name. It does not describe what the function does. Can you try to find a better name ?